dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -1971,6 +1975,35 @@ fn print_task_inventory_collection(details: &serde_json::Value) {
     };
 }
 
+fn print_task_inventory_load(details: &serde_json::Value) {
+    match serde_json::from_value::<InventoryLoadStatus>(details.clone()) {
+        Err(error) => eprintln!(
+            "warning: failed to interpret task details: {:?}: {:?}",
+            error, details
+        ),
+        Ok(status) => match status {
+            InventoryLoadStatus::Error(error) => {
+                println!("    task did not complete successfully: {error}");
+            }
+            InventoryLoadStatus::NoCollections => {
+                println!("    no collections available to load");
+            }
+            InventoryLoadStatus::Loaded {
+                collection_id,
+                time_started,
+                time_loaded,
+            } => {
+                println!(
+                    "    loaded latest inventory collection as of {}: \
+                         collection {collection_id}, taken at {}",
+                    humantime::format_rfc3339_millis(time_loaded.into()),
+                    humantime::format_rfc3339_millis(time_started.into()),
+                );
```

Comment:
turbo nit: looking at the expectorate output, this ends up formatting a pretty long line, i might consider splitting it so it's less likely to wrap in smallish terminals?

maybe something like:
```suggestion
                println!(
                    "    loaded latest inventory collection as of {}:\n\
                           collection {collection_id}\n\
                           taken at {}",
                    humantime::format_rfc3339_millis(time_loaded.into()),
                    humantime::format_rfc3339_millis(time_started.into()),
                );
```